deployment_report/templates/viya_deployment_report.html.j2

```diff
@@ -130,6 +130,26 @@
   </div>
   {# Cluster Overview: Kubernetes Versions Accordion #}
 
+  {# Cluster Overview: Storage Accordion #}
+  {% set kind_name = "Storage" %}
+  <div class="jq-accordion">
+    <h2>Storage</h2>
+    <div>
+      {% for db, info in report_data.kubernetes.dbInfo.items() %}
+      <h7>Database({{ db }})</h7>
```

Comment:
Just a space needed between "Database" and the parenthetical: `<h7>Database ({{ db }})</h7>`.